.github/workflows/tag-iteration-ends.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - uses: actions/checkout@v3
       - name: Get last date of iteration
         id: date
-        run: echo "::set-output name=date::$(date -u --date yesterday +'%m%d')"
+        run: echo "name=date=$(date -u --date yesterday +'%m%d')" >> $GITHUB_OUTPUT
```

Comment:
This has been put in place to handle the warning with "set-output" being deprecated module.